documentation/staging/content/userguide/managing-operators/preparation.md

```diff
@@ -86,25 +62,9 @@ The operator Helm chart includes:
 - Helm configuration value settings for fine tuning operator behavior.
 - Commands for deploying (installing) or undeploying the operator.
 
-You can set up access to the operator Helm chart using either a _file-based approach_
-or a _GitHub chart repository approach_:
-
-##### _File-based approach_
```

Comment:
This is still a valid approach that should be documented, but is rarely used. The write-up should clearly de-emphasize it. Maybe at least move this section _after_ the more common "repo" approach.   Also rework the introductory sentence to something like 
You can set up access to the operator Helm chart using either a _GitHub chart repository approach_ (the most common approach) or a _file-based approach_ (rarely needed):

Reply:
@tbarnes-us From this Slack conversation, https://proddev-paas-fmw.slack.com/archives/C0359AAAK8U/p1646173237826519, I've concluded that we don't need to document the rarely-needed, file-based approach. Certainly, if it becomes an issue for our customers, I will re-evaluate this decision.

Reply:
@rosemarymarano I think a case can be made for the need to cover the "how to download the helm chart if Internet access is not available" but both Ryan and I would prefer to find a way to not pollute the main flow of the docs that new users are reading with this obscure use case.  Not sure how best to accomplish that.

@tbarnes-us What I feel should be documented for this "how to download the helm chart if Internet access is not available" section is how to use `helm pull` to download the chart and move it to the machine with no Internet access so that the user can run `helm install`.  I shouldn't need to clone the source code repo to accomplish this task.  Cloning the source code repo to get access to the Helm chart could have undesirable side effects since the latest Helm chart in `main` may or may not work correctly for a production use case with a release version of operator.

Reply:
@robertpatrick ok, I will figure something out for this information and put it into an upcoming commit. Thank you.